wifiphisher/common/accesspoint.py

```diff
@@ -59,52 +139,48 @@ def start_dhcp_dns(self):
         proc = subprocess.check_output(['ifconfig', str(self.interface)])
         if constants.NETWORK_GW_IP not in proc:
             return False
-        subprocess.call(
-            ('route add -net %s netmask %s gw %s' %
-             (constants.NETWORK_IP, constants.NETWORK_MASK, constants.NETWORK_GW_IP)),
-            shell=True)
-
+        subprocess.call(('route add -net %s netmask %s gw %s' %
+                         (constants.NETWORK_IP, constants.NETWORK_MASK,
+                          constants.NETWORK_GW_IP)),
+                        shell=True)
 
     def start(self):
-        config = (
-            'interface=%s\n'
-            'driver=nl80211\n'
-            'ssid=%s\n'
-            'hw_mode=g\n'
-            'channel=%s\n'
-            'macaddr_acl=0\n'
-            'ignore_broadcast_ssid=0\n'
-        )
+        """
+        Start the softAP
+        :param self: An AccessPoint object
+        :type self: AccessPoint
+        :return: None
+        :rtype: None
+        """
+
+        # create the configuration for roguehostapd
+        hostapd_config = {
+            "ssid": self.essid,
+            "interface": self.interface,
+            "channel": self.channel}
```

Comment:
Yes. I haven't enabled the KARMA option for lunching access point since it should be passed as an option in the future. 
we can add one item `"karma_enable: 1"` in the `hostapd_config` dictionary. Like the following:

```python
hostapd_config = {
    "ssid": self.essid,
    "interface": self.interface,
    "channel": self.channel,
    "karma_enable": 1
}
```